projects/Objects/HomogenousTree/src/com/valentin/HTree.java

```diff
@@ -0,0 +1,144 @@
+package com.valentin;
+
+
+import java.util.Vector;
+
+public class HTree {
+    Vector<Node> nodesVector=new Vector();
+    Node rootNode=new Node();
+    boolean flagContains;
+
+    public HTree(){}
+
+    /**
+     *
+     * @param object
+     * @param value
+     */
+    public void add(Object object,int value){
```

Comment:
It's not clear which param is the key and which is the value. It will be more readable if you use something like: add(int key, Object value) (sorted by key, and node has object value)